docs/src/main/sphinx/release/release-388.md

```diff
@@ -0,0 +1,44 @@
+# Release 388 (29 June 2022)
+
+## General
+
+* Add support for `EXPLAIN (TYPE LOGICAL, FORMAT JSON)`. ({issue}`12694`)
+* Add `use_exact_partitioning` session property to re-partition data when the
+  upstream stage's partitioning does not exactly match what the downstream stage
+  expects. ({issue}`12495`)
+* Improve read performance for `ROW` data types. ({issue}`12926`)
+* Remove the grouped execution mechanism, including the
+  `grouped-execution-enabled`, `dynamic-schedule-for-grouped-execution`,
+  and `concurrent-lifespans-per-task` configuration properties and the
+  `grouped_execution`, `dynamic_schedule_for_grouped_execution`, and
+  `concurrent_lifespans_per_task` session properties.
+
+## Security
+
+* Add [refresh token](https://oauth.net/2/refresh-tokens/) support in OAuth 2.0. ({issue}`12664`)
+
+## Delta Lake connector
+
+* Add support for setting table and column comments with the `COMMENT`
+  statement. ({issue}`12971`)
+* Support reading tables with the property `delta.columnMapping.mode=name`. ({issue}`12675`)
+* Allow renaming tables with an explicitly set location. ({issue}`11400`)
```

Comment:
Wouldn't saying "external table" be clearer? cc @findepi

Reply:
I'm honestly not sure what the explicit location means - the PR just says it's adding support for `ALTER TABLE RENAME TO` and doesn't mention any caveats.